website/pages/docs/k8s/connect/ingress-gateways.mdx

```diff
@@ -5,4 +5,203 @@ sidebar_title: Ingress Gateways
 description: Configuring Ingress Gateways on Kubernetes
 ---
 
-# Ingress Gateways on Kubernetes
\ No newline at end of file
+# Ingress Gateways on Kubernetes
+
+-> 1.8.0+: This feature is available in Consul versions 1.8.0 and higher
+
+~> This topic requires familiarity with [Ingress Gateways](/docs/connect/ingress-gateway).
+
+This page describes how to enable external access to Connect service mesh services running inside Kubernetes using Consul ingress gateways.
+See [Ingress Gateways](/docs/connect/ingress-gateway) for more information on use-cases and how it works.
+
+Adding an ingress gateway is a multi-step process that consists of the following steps:
+
+* Setting the helm chart configuration
+* Deploying the helm chart
+* Configuring the gateway
+* Defining an Intention (if ACLs are enabled)
+* Deploying your application to Kubernetes
+* Connecting to your application
+
+## Setting the helm chart configuration
+When deploying the helm chart you must provide helm with a custom yaml file that contains your environment configuration.
+
+```yaml
+global:
+  name: consul
+connectInject:
+  enabled: true
+ingressGateways:
+  enabled: true
+  gateways:
+    - name: ingress-gateway
+      replicas: 2
+      service:
+       type: LoadBalancer
+```
+~> *Note:* this will create a public unathenticated LoadBalancer in your cluster, please take appropriate security considerations.
+
+The yaml snippet is the launching point for a valid configuration that must be supplied when installing using the [official consul-helm chart](https://hub.helm.sh/charts/hashicorp/consul).
+Information on additional options can be found in the [Helm reference](/docs/k8s/helm). Configuration options for ingress gateways reside under the [ingressGateways](/docs/k8s/helm#v-ingressgateways) entry.
+
+
+The gateways stanza is where you will define and configure the set of ingress gateways you want deployed to your environment.
+The only required field for each entry is `name`, though entries may contain any of the fields found in the `defaults` stanza.
+Values in this section override the values from the defaults stanza for the given ingress gateway with one exception:
+the annotations from the defaults stanza will be *appended* to any user-defined annotations defined in the gateways stanza rather than being overridden.
+Please refer to the ingress gateway configuration [documentation](/docs/k8s/helm#v-ingressgateways-defaults) for a detailed explanation of each option.
+
+-> *Note*: Make sure any ports that will be used as listeners in the ingress gateway's Consul config entry are included in the `ports` object for each gateway.
```

Comment:
It could be helpful to mention that ports 8080 and 8443 are exposed for traffic by default.